### PR TITLE
[1.40.x] Release Notes for 1.40.0.Final

### PR DESCRIPTION
**Pull request by ricardozanini**
Release Notes for 1.40.0.Final

### CODE REVIEW

**Review by cristianonicolai — APPROVED** [review]

**Comment by spolti** [comment]
Just one observation, is not yet ready, Cristina brought up a small improvement on the quarkus extension side, now we need to apply it in the image side.

**Comment by radtriste** [comment]
> Just one observation, is not yet ready, Cristina brought up a small improvement on the quarkus extension side, now we need to apply it in the image side.

@spolti This will be for a 1.40.1

**Comment by radtriste** [comment]
@cristianonicolai @ricardozanini @MarianMacik 
I went through:
- https://github.com/kiegroup/kogito-runtimes/releases/tag/1.40.0.Final
- https://github.com/kiegroup/kogito-apps/releases/tag/1.40.0.Final
- https://github.com/kiegroup/kogito-examples/releases/tag/1.40.0.Final

Here is the result: https://github.com/kiegroup/kogito-docs/pull/372

**We need to automate this and ask people to be more rigorous. Many PRs were not resolved and/or no final version there ... I updated them.**

**Comment by radtriste** [comment]
Once Cloud release is done, I will provide the update with the corresponding issues

**Review by MarianMacik — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -214,19 +214,25 @@ More details at [Antora documentation](https://docs.antora.org/antora/latest/asc
 
 ## Generating Release Notes for Serverless Workflow
 
-1. Use the [JIRA to search for the issues](https://issues.redhat.com/issues/?jql=) with:
+1. You can retrieve all changes in the release page of each repository of the project:
```
Comment (kaldesai) [inline CONTRIBUTING.md:217]:
```suggestion
1. You can retrieve all changes on the release page of each repository of the project:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -214,19 +214,25 @@ More details at [Antora documentation](https://docs.antora.org/antora/latest/asc
...
-project = Kogito and fixVersion = <version> and component in ("Serverless Workflow Editor", "Serverless Workflow Engine") AND status in (Resolved, Done)  and type != Sub-task and type != Epic
-```
+   - https://github.com/kiegroup/kogito-runtimes/releases/tag/{version}
+   - https://github.com/kiegroup/kogito-apps/releases/tag/{version}
+   - https://github.com/kiegroup/kogito-examples/releases/tag/{version}
+   - https://github.com/kiegroup/kogito-runtimes/releases/tag/{version_without_Final}
+   - https://github.com/kiegroup/kogito-runtimes/releases/tag/v{version_without_Final}
 
-Replace `<version>` with the given version, for example `1.35.0.Final`.
+Replace `{version}` with the given core version, for example `1.40.0.Final`.
```
Comment (kaldesai) [inline CONTRIBUTING.md:225]:
```suggestion
Replace `{version}` with the given core version, for example, `1.40.0.Final`.  
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -214,19 +214,25 @@ More details at [Antora documentation](https://docs.antora.org/antora/latest/asc
...
-```
+   - https://github.com/kiegroup/kogito-runtimes/releases/tag/{version}
+   - https://github.com/kiegroup/kogito-apps/releases/tag/{version}
+   - https://github.com/kiegroup/kogito-examples/releases/tag/{version}
+   - https://github.com/kiegroup/kogito-runtimes/releases/tag/{version_without_Final}
+   - https://github.com/kiegroup/kogito-runtimes/releases/tag/v{version_without_Final}
 
-Replace `<version>` with the given version, for example `1.35.0.Final`.
+Replace `{version}` with the given core version, for example `1.40.0.Final`.  
+Replace `{version_without_Final}` with the given cloud version, for example `1.40.0`.
```
Comment (kaldesai) [inline CONTRIBUTING.md:226]:
```suggestion
Replace `{version_without_Final}` with the given cloud version, for example, `1.40.0`.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -214,19 +214,25 @@ More details at [Antora documentation](https://docs.antora.org/antora/latest/asc
...
 
 2. Update the page [release_notes.adoc](serverlessworkflow/modules/ROOT/pages/release_notes.adoc)
-3. Align with the team what should be under "Notable changes"
-4. Add the rest to "Other Changes and Bug Fixing".
-5. Open a PR in the target **branch** version, **not** main
-6. Add one member from each squad to review
+
+   You should get JIRA issues with `KOGITO` and `DROOLS` projects as well as issues coming from the `kie-issues` repository.
```
Comment (kaldesai) [inline CONTRIBUTING.md:230]:
```suggestion
You must get JIRA issues with the `KOGITO` and `DROOLS` projects and issues from the `kie-issues` repository.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -214,19 +214,25 @@ More details at [Antora documentation](https://docs.antora.org/antora/latest/asc
...
+
+4. Align with the team what should be under "Notable changes"
+5. Add the rest to "Other Changes and Bug Fixing".
+6. Open a PR in the target **branch** version, **not** main
```
Comment (kaldesai) [inline CONTRIBUTING.md:234]:
```suggestion
6. Open a PR in the target **branch** version, **not** main.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -214,19 +214,25 @@ More details at [Antora documentation](https://docs.antora.org/antora/latest/asc
...
+7. Add one member from each squad to review
```
Comment (kaldesai) [inline CONTRIBUTING.md:235]:
```suggestion
7. Add one member from each squad to review.
```

**Review by kaldesai — APPROVED** [review]
@ricardozanini, Minor suggestions. Otherwise, your content looks good to me.

**Pull request merged**